mods/ts/sequences/civilian.yaml

```diff
@@ -862,6 +862,27 @@ gakodk:
 	critical-idle:
 		Start: 2
 		ShadowStart: 5
+	large-lights: gakodk_a
+		Length: 12
+		Tick: 200
+	damaged-large-lights: gakodk_a
+		Start: 12
+		Length: 12
+		Tick: 200
+	small-light: gakodk_b
+		Length: 12
+		Tick: 200
+	damaged-small-light: gakodk_b
```

Comment:
This didn't existed in TS (damage variant disabled this particular anim), but I guess it can stay - the frames there are empty.